src/index.ts

```diff
@@ -330,8 +330,8 @@ function main() {
                         answers.dockerTag = program.dockerTagOverride;
                     } else {
                         // For a released version the docker tag and version should be same
-                        // Default to latest released verion
-                        const version = '1.0.0';
+                        // Default to latest released verion (different for remotemonitoring and devicesimulation)
+                        const version = (program.type === 'remotemonitoring') ? '1.0.0' : 'DS-1.0.2';
```

Comment:
I would change the `program.type === 'devicesimulation'` and default to 1.0.0. Even though this works right now, we need a better solution to query latest release from github per solution type and use that version number to avoid changing the default value each release. Can you please open an issue that can track this?

Reply:
There are 2 types for solution types for simulation - 'devicesimulation' and 'devicesimulation-nohub'. So it is simpler to check if it is not 'remotemonitoring'. This is basically for Dev's ease, we are frequestly using CLI for test deployments and it is annoying to add override params everytime. I will create an issue to have simpler solution

Reply:
Something that will help ease dev's pain point is to use all the options through pcs .... We have saved the different options in launch.json in vscode. This makes the deployment essentially one click.

Also I think solution type should have been `devicesimulation` but supporting different skus. Obviously this will be a little bit of change, but something to consider for your backlog.